testing/src/main/java/io/stargate/it/grpc/ExecuteQueryTest.java

```diff
@@ -81,4 +83,49 @@ public void simpleQueryWithPaging() throws InvalidProtocolBufferException {
     assertThat(rs.getPagingState()).isNotNull();
     assertThat(rs.getPageSize().getValue()).isGreaterThan(0);
   }
+
+  @Test
+  public void useKeyspace(@TestKeyspace CqlIdentifier keyspace) {
```

Comment:
nit, I think that it can be split into 3 test cases:
- use keyspace
- query without a keyspace
- query with a keyspace
Lower granularity tests give better feedback. wdyt?

Reply:
@ivansenic @tomekl007 It's not three different uses case though. It verifying that state doesn't persist between request when using the same stub/channel.